xtos/include/sof/lib/perf_cnt.h

```diff
@@ -25,6 +25,8 @@ struct perf_cnt_data {
 	uint32_t cpu_delta_peak;
 	uint32_t cpu_delta_sum;
 	uint32_t sample_cnt;
+	/* indicate which period have peak mcps in current module */
+	uint32_t peak_mcps_cnt;
```

Comment:
This variable name suggests that we count peak_mcps occurrence. While there is a comment that explain actual meaning the name is still a bit confusing. Maybe you could change it to something like 'peak_mcps_sample' ?